esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -559,7 +559,7 @@ define([
         refreshActionState: function () {
             var isArchived = this.wu.get("Archived");
             this.setDisabled(this.id + "AutoRefresh", isArchived || this.wu.isComplete(), "iconAutoRefresh", "iconAutoRefreshDisabled");
-            registry.byId(this.id + "Save").set("disabled", isArchived || !this.wu.isComplete() || this.wu.isDeleted());
+            registry.byId(this.id + "Save").set("disabled", isArchived || !this.wu.isComplete() || this.wu.isDeleted() || !this.wu.isBlocked());
```

Comment:
Is the goal to enable the save button while the WU is blocked but incomplete?  If so should it be:
isArchived || (!this.wu.isComplete() && !this.wu.isBlocked()) || this.wu.isDeleted()
?